src/components/fields/LegendgroupCreator.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react';
+import {connectToContainer} from 'lib';
+import Field from './Field';
+import Dropdown from './Dropdown';
+import PropTypes from 'prop-types';
+import Button from '../widgets/Button';
+import {PlusIcon} from 'plotly-icons';
+
+class UnconnectedLegendgroupCreator extends Component {
```

Comment:
Let's generalize this into a `GroupCreator` with a `prefix` attr so we can immediately reuse it for `stackgroup`